src/QueueManager.php

```diff
@@ -218,7 +218,7 @@ public static function push($callable, array $args = [], array $options = []): v
         $message = new ClientMessage([
             'queue' => $queue,
             'class' => $callable,
-            'args' => [$args],
+            'args' => $args,
```

Comment:
I agree that this wrapping array is a bit redundant. However, I'm concerned that there isn't a safe way to remove this without having duplicate data in the messages for at least a few releases.

Because `$args` can also be a positional list and have only one value eg. `QueueManager::push(WelcomeMailer::class, [1]);`. This scenario being functional right now, means we cannot reuse `args` and have a zero-downtime upgrade process.